.circleci/config.yml

```diff
@@ -318,8 +272,11 @@ workflows:
   version: 2.1
   build:
     when:
-      not:
-        equal: [ main, << pipeline.git.branch >> ]
+      and:
+        - not:
+            equal: [ main, << pipeline.git.branch >> ]
+        - not:
+            matches: { pattern: "^ghactionsonly-.*", value: << pipeline.git.branch >> }
```

Comment:
Why not simply leave this away to ensure the remaining tests run OK -- until we move completely away from CCI?

Reply:
Lines 276-277 were already there (though the PR touches them because 275 and 278-279 were added).  I added 278-279 so that I could iterate on the Github Actions tests without triggering tests on the other systems.  I guess in principle I could remove that now for merging, but it also seemed like it could be useful to have in the future. I'm fine either way.